CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 ## Unreleased
 
+### Added
+
+[#2664 ](https://github.com/thanos-io/thanos/pull/2664) Website: Search box functionality on the Thanos website.
```

Comment:
No need for that (: This is not a feature of Thanos components, but our amazing website :muscle:

Reply:
Alright. would fix that now.
Thanks for pointing that out